instance/tests/test_tasks.py

```diff
@@ -77,7 +82,38 @@ def test_watch_pr_new(self, mock_get_username_list, mock_get_pr_list_from_userna
         self.assertEqual(instance.fork_name, 'watched/fork')
         self.assertEqual(instance.github_pr_number, 234)
         self.assertEqual(instance.branch_name, 'watch-branch')
-        self.assertEqual(instance.ansible_extra_settings, 'WATCH: true\r\n')
+        self.assertEqual(instance.database_url, 'mysql://db.opencraft.com')
+        self.assertEqual(instance.mongo_url, 'mongo://mongo.opencraft.com')
+        self.assertEqual(yaml.load(instance.ansible_extra_settings), {'WATCH': True})
```

Comment:
That answers @smarnach 's question in https://github.com/open-craft/opencraft/pull/11/files#r41064245 - I'm not sure I like this additional magic, of removing variables from the user defined `ansible_extra_settings`. Can we instead use the order of precedence that comes from `ANSIBLE_SETTINGS` to determine which value we end up passing to ansible? https://github.com/open-craft/opencraft/pull/11/files#diff-848aab5153020ca3817d6db9d4d3a144R535